src/ui/preview/Preview.vue

```diff
@@ -156,9 +156,7 @@ export default {
       delete this.existingViewElement;
     },
     initializeViewContainer() {
-      this.viewContainer = document.createElement('div');
-      this.viewContainer.classList.add('l-angular-ov-wrapper');
```

Comment:
Same question about the class here

Reply:
No longer needed according to @charlesh88